source/class/cv/ui/manager/dialog/ValidationError.js

```diff
@@ -0,0 +1,174 @@
+/**
+ * Show validation errors for a config file.
+ */
+qx.Class.define('cv.ui.manager.dialog.ValidationError', {
+  extend: dialog.Dialog,
+
+  /*
+  ***********************************************
+    CONSTRUCTOR
+  ***********************************************
+  */
+  /**
+   *
+   * @param file {cv.ui.manager.model.FileItem}
+   * @param content {String}
+   * @param errors {Array<Map>}
+   * @param properties {Map?}
+   */
+  construct: function (file, content, errors, properties) {
+    this._file = file;
+    this._content = content.split("\n");
+    this._errors = errors;
+    this.base(arguments, Object.assign(properties || {}, {
+      useBlocker: true
+    }));
+
+    this.addListener("appear", () => {
+      if (!window.monaco) {
+        cv.ui.manager.editor.Source.load(this.highlight, this);
+      } else {
+        this.highlight();
+      }
+    })
+  },
+
+  /*
+  ***********************************************
+    EVENTS
+  ***********************************************
+  */
+  events: {
+   action: "qx.event.type.Data"
+  },
+
+  /*
+  ***********************************************
+    MEMBERS
+  ***********************************************
+  */
+  members: {
+    _content: null,
+    _errors: null,
+    /**
+     * @var {cv.ui.manager.model.FileItem}
+     */
+    _file: null,
+
+    highlight: function () {
+      Array.from(document.querySelectorAll('pre.highlight')).forEach (elem => {
+        monaco.editor.colorizeElement(elem, {
+          theme: 'vs-dark'
+        });
+      });
+    },
+
+    /**
+     * Create the main content of the widget
+     */
+    _createWidgetContent: function() {
+      let container = this._createDialogContainer();
+      let vbox = new qx.ui.container.Composite(new qx.ui.layout.VBox(10));
+      container.add(vbox);
+
+      let prologue = this.tr('<h3>Config Errors</h3>\
+<p>The validation of "%1" showed some errors. It is NOT recommended to edit this file in the XML-Tree editor, because \
+if can break the file completely. Please open this file in the text editor and fix the errors there. \
```

Comment:
"because if can break" -> "because it can break"

Reply:
Fixed